### PR TITLE
Added section on "Running simulations"

### LINKED ISSUE #144: Page on running Slurm job
Common issues like
* networking
* hostfiles
* grouping executables
* checking failure states
* multiple mpirun calls

### PR DESCRIPTION
**Pull request by fsimonis**
This PR:
* Adds a new section "Running simulations"
* Adds a super basic explanation of how to do this locally
* Moves the page on output files to the new section
* Adds guides for distributed systems and SLURM-managed clusters

Closes #144

### CODE REVIEW

File: _data/sidebars/docs_sidebar.yml [hunk 1]
```diff
@@ -437,6 +433,27 @@ entries:
...
 
+  - title: Running simulations
+    output: web, pdf
+    folderitems:
+
+    - title: Overview
+      url: /running-overview.html
+      output: web, pdf
+      type: homepage
+
```
Comment (uekerman) [inline _data/sidebars/docs_sidebar.yml:444]: Aren't you missing the "running simple" page here?

File: pages/docs/running/running-simple.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+title: Running on a local machine
+permalink: running-simple.html
+keywords: running, mpi, local, workstation, laptop
+summary: "Run your preCICE simulation on your local machine."
+---
+
+## Preparation
+
+Before we run the simulation, lets check some very common pitfalls.
```
Comment (uekerman) [inline pages/docs/running/running-simple.md]:
```suggestion
Before we run the simulation, let's check some very common pitfalls.
```

File: pages/docs/running/running-simple.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+ cd /path/to/solver/a
+ ./runSolver &
+
+ # Launch solver B
+ cd /path/to/solver/b
+ ./runSolver &
+
+ # Launch solver C
+ cd /path/to/solver/c
+ ./runSolver &
```
Comment (uekerman) [inline pages/docs/running/running-simple.md]:
```suggestion
 # Launch solver A
 cd /path/to/solver/a
 ./runSolver &> a.log &

 # Launch solver B
 cd /path/to/solver/b
 ./runSolver &> b.log &

 # Launch solver C
 cd /path/to/solver/c
 ./runSolver  &> c.log &
```

File: pages/docs/running/running-simple.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+ # Wait for every solver to finish
+  wait
+)
+```
+
+## Next steps
+
+- See which [output-files preCICE](running-output-files) creates.
```
Comment (uekerman) [inline pages/docs/running/running-simple.md]:
```suggestion
- See which [output-files](running-output-files) preCICE creates.
```

File: pages/docs/running/running-distributed.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+   A distributed system requires communication between nodes using a common network.
+2. The local system has unhindered access to the local files.
+   A distributed system requires to use a shared network drive with higher latency.
+
+So, there is additional work involved to set the simulation up.
+
+## Common network interface
+
+preCICE defaults to the loopback interface which will only work on the local system.
+Follow the [instructions on how to configure the communication](configuration-communication)
```
Comment (uekerman) [inline pages/docs/running/running-distributed.md]: Maybe mention here again that this concerns the communication. And specifically only "sockets", right?

File: pages/docs/running/running-slurm.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+```bash
+# Distributing 3 nodes of 24 tasks each to 3 participants
+# One node per participant.
+sed -n " 1,24p" hosts.ompi > hosts.a 
+sed -n "25,48p" hosts.ompi > hosts.b
+sed -n "49,72p" hosts.ompi > hosts.c 
+```
+
+{% warning %}
+Hostfiles are not standardized and differ between OpenMPI, MPICH and IntelMPI!
```
Comment (uekerman) [inline pages/docs/running/running-slurm.md]:
```suggestion
Hostfiles are not standardized and differ between OpenMPI, MPICH and IntelMPI.
```

**Review by uekerman — APPROVED** [review]
Looking good :+1: 
I guess it would be good to add a few links from other pages to theses ones. And also from here to other pages.

File: pages/docs/running/running-distributed.md (hunk 5 above)
Comment (fsimonis) [reply pages/docs/running/running-distributed.md]: This is explained on the linked communication page. I'll make it explicit here.

**Pull request merged**